Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mozillamarketplace/centos-mysql-mkt:0.2
+
+RUN yum install -y supervisor
+
+ENV IS_DOCKER 1
```

Comment:
you can remove this line, we are trying to remove it from marketplace-env Dockerfile where you probably copied it from